extensions/sql-migration/src/wizard/wizardController.ts

```diff
@@ -48,23 +47,41 @@ export class WizardController {
 
 		this._wizardObject.generateScriptButton.enabled = false;
 		this._wizardObject.generateScriptButton.hidden = true;
-		const saveAndCloseButton = azdata.window.createButton(loc.SAVE_AND_CLOSE);
-		this._wizardObject.customButtons = [saveAndCloseButton];
+		this._wizardObject.nextButton.position = 'left';
+		this._wizardObject.nextButton.secondary = false;
+		this._wizardObject.doneButton.label = loc.START_MIGRATION_TEXT;
+		this._wizardObject.doneButton.position = 'left';
+		this._wizardObject.doneButton.secondary = false;
+		this._wizardObject.backButton.position = 'left';
+		this._wizardObject.backButton.secondary = true;
+		this._wizardObject.cancelButton.position = 'left';
+		this._wizardObject.cancelButton.secondary = true;
+
+		const saveAndCloseButton = azdata.window.createButton(
+			loc.SAVE_AND_CLOSE,
+			'right');
+		saveAndCloseButton.secondary = true;
+
+		const validateButton = azdata.window.createButton(
```

Comment:
I like that we added this, but I noticed on my first run through the new experience that it wasn't super intuitive to find the validation dialog. Maybe it's just me but I guess I expected this as a separate step in the wizard instead of a button in the bottom left of the wizard, only on this one page.

Is there any way we could somehow bring more attention to this? Whether it's by maybe changing the color of the button, or the position, or even just by simply calling out that it exists (e.g. somewhere on the page, maybe at the bottom, just add a small text that suggests that the user attempts a validation before proceeding in the wizard).

+ @croblesm

Reply:
Somewhat related to this, I also feel that this button which launches the validation dialog should be disabled by default, and only enabled once all the fields on this page are filled in. 

As soon as I land on this page (and all the input fields are still empty) I already have the option to run the validation. And if I do, it will instantly fail since I haven't provided any of the actual details yet which the validation is supposed to be checking (such as the network share path for the source databases):

![image](https://user-images.githubusercontent.com/11844501/207183843-32b6955c-da91-4caa-8ffc-51c61b1007b4.png)

![image](https://user-images.githubusercontent.com/11844501/207183865-7f4144a5-8e25-4d62-a30d-59669b7c337f.png)

So I think it would make sense if the validation stayed disabled until all the required fields are provided.

Reply:
Now that I think about it, this might be related to my earlier comment about 400 errors since I saw a very similar error code there: https://github.com/microsoft/azuredatastudio/pull/21386#discussion_r1046488991

Reply:
Great feedback @raymondtruong. I will try to repro this bug in my environment.